### PR TITLE
Page  branch

### PR DESCRIPTION
**Pull request by deikdesign**
- Adding pages to the website
- Using JavaScript to hide and show sections of the page
- Use Bootstrap to give style to the page
- Add date and time to the page

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -93,3 +98,72 @@ function removeB() {
...
+  const id = ['days', 'monthly', 'daynumb', 'years', 'hours', 'minutes', 'seconds', 'ampm'];
+  const values = [week[daname], months[mont], denum, yrr, hourr.pa(2), min.pa(2), sec, pe];
+
+  for (let i = 0; i < id.length; i += 1) { document.getElementById(id[i]).firstChild.nodeValue = values[i]; }
+}
+
+function initClock() {
+  updateClock();
+  // eslint-disable-next-line no-implied-eval
+  window.setInterval('updateClock()', 1);
```
Comment (cyonii) [inline script.js]: - [x] [OPTIONAL] Is there a reason why you have passed the callback to `setInterval` as a string? I tried to pass it like `window.setInterval(updateClock, 1);` and it seems to work fine. You don't need to call the callback you pass to `setInterval`, it knows how to handle it, so please consider the approach I showed.

File: script.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 /* eslint-disable no-unused-vars */
 /* eslint-disable max-classes-per-file */
+/* eslint-disable max-len */
+/* eslint-disable no-undef */
+/* eslint-disable no-implied-eval */
+/* eslint-disable no-var */
+/* eslint-disable no-extend-native */
```
Comment (cyonii) [inline script.js]: - [x] Disabling these rules defeats the reason why we have the linters configured in the first place. Please do not disable linter rules, there are there to guide you to write your code according to the specified standard. Remove the disabled rules and follow the standards specified by the linters.

**Review by cyonii — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review to see the changes you need to make._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (deikdesign) [reply script.js]: Yes, you are right. I changed it.

File: script.js (hunk 2 above)
Comment (deikdesign) [reply script.js]: Issues are fixed

**Comment by helman101** [comment]
# STATUS: APPROVED 🟢 🥳 

Hi @deikdesign & @danalvrz 🥇 , 

### Congratulations!!! 🥳 🎉

You have successfully completed all the expectations for this project, there is nothing more to do here, now you can go on your way and keep improving your skills.

### Optional Changes 🤓 

- [ ] [OPTIONAL] I will kindly ask you to delete all the commented code, so your project looks cleaner 😄 

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**